http-clients/src/test/resources/service-config-example.yml

```diff
@@ -0,0 +1,73 @@
+auth:
+  service: # just a standard ServiceConfig block
+    security:
+      trustStorePath: src/test/resources/trustStore.jks
+    uris:
+      - https://some.internal.url:8643/service/api
+  oauth2:
+    id: example
+    secret: example-secret
+    # may also be the full URL (https://some.url:8643/auth/api), but just path preferred
+    endpoint: /auth/api
+    # may also be the full URL (https://this.app.dev:port/<contextPath>/redirect), but just path preferred
+    clientRedirectEndpoint: /example/redirect
+  legacyModeEnabled: false
+
+serviceDiscovery:
+  # default security configuration, to be used when more specific configurations
+  # are not applied at a per-service level
+  security:
+    trustStorePath: src/test/resources/trustStore.jks # mandatory
+    trustStoreType: JKS                               # optional (defaults to JKS)
+    keyStorePath: src/test/resources/keyStore.jks     # optional
+    keyStorePassword: keystore                        # mandatory iff keyStorePath defined
+    keyStoreKeyAlias: test                            # mandatory iff keyStorePath defined
+    keyStoreType: JKS                                 # optional (defaults to JKS)
+
+  # optional API token
+  apiToken: defaultApiDToken
+
+  services:
+    hello:
+      # list of service API endpoints (mandatory)
+      uris:
+        - https://localhost:8443/example/api
+      # optional API token
+      apiToken: helloApiToken
+      # optional local security block (uses existing SslConfiguration spec)
+      security:
+        trustStorePath: src/test/resources/trustStore.jks # mandatory if security block included
+        keyStorePath: src/test/resources/keyStore.jks     # optional
+        keyStorePassword: keystore                        # mandatory iff keyStorePath defined
+        keyStoreKeyAlias: test                            # mandatory iff keyStorePath defined
+        keyStoreType: JKS                                 # optional (defaults to JKS)
+
+    goodbye: # just a standard ServiceConfig block
+      security:
+        trustStorePath: src/test/resources/trustStore.jks
+      uris:
+        - https://localhost:8443/example/api
+
+# information for the front-end
+clients:
```

Comment:
just an example for the test and shows a working version of the config similar to the real world config